tests/test_enterprise/api/test_views.py

```diff
@@ -5792,6 +5835,128 @@ def test_enroll_learners_in_courses_partial_failure(self, mock_get_course_mode,
         self.assertEqual(response.status_code, status.HTTP_409_CONFLICT)
         self.assertEqual(response.json(), enrollment_response)
 
+    @ddt.data(
+        # Creates DefaultEnterpriseEnrollmentRealization upon successful subscription
+        # enrollment (is_default_auto_enrollment is True)
+        {
+            'body': {
+                'notify': 'true',
+                'enrollments_info': [
+                    {
+                        'email': 'abc@test.com',
+                        'course_run_key': fake_catalog_api.FAKE_COURSE_RUN.get('key'),
+                        'mode': VERIFIED_COURSE_MODE,
+                        'license_uuid': '2aa06d86-e2bf-40b8-81e7-b0449fa5541f',
+                        'is_default_auto_enrollment': True,
+                    },
+                ]
+            },
+            'fulfillment_source': LicensedEnterpriseCourseEnrollment,
+            'expected_enrollment_realization_count': 0,
```

Comment:
Shouldn't this test create a new realization?

Reply:
D'oh, you're correct! This wasn't working properly as I had not added `is_default_auto_enrollment` to `EnrollmentsInfoSerializer`, so the field wasn't including during processing, always falling back to `is_default_auto_enrollment=False`.

Updated/fixed via https://github.com/openedx/edx-enterprise/pull/2276/commits/24886e1522fcf43fd70642b2dfd989b2a77ca710